src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menInfo = people.filter(person => {
```

Comment:
this is not info

menInfo means an object with men information, u have array here
```suggestion
  const men = people.filter(person => {
```